### DISCUSSION TITLE
Adding Horizontal Scaling as Improvement

### DISCUSSION THREAD

**Discussion by aounalazzam** [pocketbase/pocketbase#4703] [Polls]
the disadvantage of Pocketbase is still a vertical scale that is not recommended for extreme scaling but I have an idea to make that can add first before writing the first admin credentials can add credentials for any SQL database or choose if needed locally then test the connection and continue that can make Pocketbase scale at a high level

**Comment by ganigeorgiev** [comment]
Sorry but there are no plans for this. And honestly I'm getting tired of people constantly complaining about scaling without providing a single metric to understand whether their use case actually needs it and why they want to use PocketBase in the first place.

In any case this was discussed several times before in https://github.com/pocketbase/pocketbase/issues/234, https://github.com/pocketbase/pocketbase/issues/528, https://github.com/pocketbase/pocketbase/discussions/615, https://github.com/pocketbase/pocketbase/discussions/1673, https://github.com/pocketbase/pocketbase/discussions/395 and probably others.

To some extend you can use [Marmot](https://github.com/pocketbase/pocketbase/discussions/487) to sync the db data between multiple PocketBase instances but it comes with a lot of caveats (see their [Limitations section](https://github.com/maxpert/marmot#limitations)) and app hooks don't work outside of their running app instance.
